testdata/p4_16_samples/issue363-bmv2.p4

```diff
@@ -0,0 +1,89 @@
+#include <v1model.p4>
+
+struct alt_t {
+  bit<1> valid;
+  bit<7> port;
+};
+
+struct row_t {
+  alt_t alt0;
+  alt_t alt1;
+};
+
+struct parsed_packet_t {};
+
+struct local_metadata_t {
+  row_t row;
+};
+
+struct macDA_t
+{
+    bit<28>   lower28Bits;
+    bit<20>   upper20Bits;
+}
+
+struct sip_t
+{
+    bit<28>	lower28Bits;
+    bit<4>	upper4Bits;
+    macDA_t     macDA;
+}
+
+header our_bitvec_header {
+    bit<2> g2;
+    bit<3> g3;
+    sip_t  sip;    
+}
+
+parser parse(packet_in pk, out parsed_packet_t hdr,
+             inout local_metadata_t local_metadata,
+             inout standard_metadata_t standard_metadata) {
+  state start {
+    transition accept;
```

Comment:
you should extract into this header to test whether this works.

Reply:
Why is this conversation marked as resolved? The extract code is still commented-out.